code/game/machinery/computer/Operating.dm

```diff
@@ -173,20 +171,29 @@
 			return FALSE
 
 /obj/machinery/computer/operating/process()
-
-	if(table && table.check_victim())
+	if(table && table.check_patient())
 		if(verbose)
-			if(patientName!=table.victim.name)
-				patientName=table.victim.name
+			if((table.patient.stat != patStatHolder) || newPat)
+				if(table.patient.stat == DEAD || table.patient.status_flags & FAKEDEATH)
+					patStat = "Dead"
```

Comment:
You are only using `patStat` inside this function. I strongly suggest making it a local variable (rather than a member variable on the computer itself).

Reply:
If it's made a local variable, the it is **created** every time `process()` is called which is on every tick.